app/controllers/api/v1/game_play_apis_controller.rb

```diff
@@ -1,8 +1,18 @@
-# require 'pry'; binding.pry
+# require './././app/serializer/game_play_serializer'
 
 class Api::V1::GamePlayApisController < ApplicationController
   def index 
-    @word = WordFacade.find_word
-    # require 'pry'; binding.pry
+    @answers = WordFacade.multi_word_search(num_param)
+    @photo = PhotoFacade.photos(@answers[0])
+    render json: GamePlaySerializer.play_response(@photo, @answers[0], @answers), status:200
+  end
+
+  private
```

Comment:
Hey, I cannot recall if there is a reason for having this be private functionality rather than functionality that takes place within the Index method itself.  Can you provide some clarification please?

Reply:
Good question! 
- My understanding and approach with using **strong params** is that they should be used in any and all instances where we **only want to be able to change those parameters for an object**. 
- In this case, **we only want to be able to change the number of words returned**, and that is based on the request which is technically hitting prior to the call. 
- So while we may not need to be this protective of the params, I also would make the argument that it doesn't hurt to do so and keep the **num_param** intact as a **strong param**. 
- Other input welcome

Reply:
your 3rd point is fair (doesn't hurt).